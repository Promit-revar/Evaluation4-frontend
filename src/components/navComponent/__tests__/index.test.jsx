import { asFragment } from '@testing-library/react';
import React from 'react';
import { render } from '@testing-library/react';
import NavComponent from '..';
describe('NavComponent', () => {
    it('should render correctly', () => {
        const { asFragment } = render(<NavComponent />);
        expect(asFragment()).toMatchSnapshot();
    });
});
