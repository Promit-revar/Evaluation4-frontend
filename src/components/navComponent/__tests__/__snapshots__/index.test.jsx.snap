// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavComponent should render correctly 1`] = `
<DocumentFragment>
  <nav>
    <div
      class="nav-container"
    >
      <div
        class="nav-logo"
      >
        <h1 />
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
